packages/travix-ui-kit/components/stepper/stepper.md

```diff
@@ -0,0 +1,31 @@
+Basic input:
+
+    <div>
+      <style dangerouslySetInnerHTML={{__html: `
+        .ui-stepper .ui-stepper__button-decrease:before {
+          content: '-';
+        }
+        .ui-stepper .ui-stepper__button-increase:before {
+          content: '+';
+        }
+      `}} />
+      <Stepper />
+    </div>
+
+Range 1 - 4 input:
+
+    <div>
+      <style dangerouslySetInnerHTML={{__html: `
+        .ui-stepper .ui-stepper__button-decrease:before {
+          content: '-';
```

Comment:
Why do we have to define it here?

Reply:
I'm using icons from RWD. We do not have them inside ui-kit demo. So I've changed content to display `+` and `-`.

Using same way, as in `dropDown/dropDown.md`